lib/b2c/m2m.ts

```diff
@@ -241,19 +241,19 @@ export class M2M {
   // ADDIMPORT: import { performAuthorizationCheck, ScopeAuthorizationFunc } from "./m2m_local";
   // ADDIMPORT: import { ClientError } from "../shared/errors";
   /**
-                   * Authenticate an access token issued by Stytch from the Token endpoint.
-                   * M2M access tokens are JWTs signed with the project's JWKs, and can be validated locally using any Stytch client library.
-                   * You may pass in an optional set of scopes that the JWT must contain in order to enforce permissions.
-                   * You may also override the default scope authorization function to implement custom authorization logic.
-                   *
-                   * @param data {@link AuthenticateTokenRequest}
-                   * @param scopeAuthorizationFunc {@link ScopeAuthorizationFunc} - A function that checks if the token has the required scopes. 
-                     The default function assumes scopes are either direct string matches or written in the form "action:resource". See the 
-                     documentation for {@link performAuthorizationCheck} for more information.
-                   * @async
-                   * @returns {@link AuthenticateTokenResponse}
-                   * @throws {ClientError} when token can not be authenticated
-                   */
+                     * Authenticate an access token issued by Stytch from the Token endpoint.
+                     * M2M access tokens are JWTs signed with the project's JWKs, and can be validated locally using any Stytch client library.
+                     * You may pass in an optional set of scopes that the JWT must contain in order to enforce permissions.
+                     * You may also override the default scope authorization function to implement custom authorization logic.
+                     *
+                     * @param data {@link AuthenticateTokenRequest}
+                     * @param scopeAuthorizationFunc {@link ScopeAuthorizationFunc} - A function that checks if the token has the required scopes. 
+                       The default function assumes scopes are either direct string matches or written in the form "action:resource". See the 
+                       documentation for {@link performAuthorizationCheck} for more information.
+                     * @async
+                     * @returns {@link AuthenticateTokenResponse}
+                     * @throws {ClientError} when token can not be authenticated
+                     */
```

Comment:
Hmmm... looks like codegen is forcing this section to slowly march right off the page. I'll file a task to look into why it's doing this.

Reply:
BACK-3893